packages/paste-website/src/components/site-wrapper/SiteHeader.tsx

```diff
@@ -29,34 +42,64 @@ export const SiteHeader: React.FC = () => {
   return (
     <Box
       as="aside"
-      backgroundColor="colorBackgroundBody"
-      borderColor="colorBorderLight"
-      borderStyle="none none solid none"
-      borderBottomWidth="borderWidth10"
-      paddingLeft="space200"
-      paddingRight="space80"
-      paddingTop="space50"
-      paddingBottom="space50"
+      display="flex"
+      alignItems="center"
+      backgroundColor="colorBackgroundPrimaryDarkest"
       position="sticky"
-      top={`${getMainContentComputedOffset(isPasteTheme)}px`}
+      top={`${getMainContentComputedOffset()}px`}
       zIndex="zIndex80"
     >
-      <Box display="flex" justifyContent="space-between" alignItems="center" flexDirection="row">
+      <Box display="flex" alignItems="center" paddingRight="space70" paddingLeft="space70" minWidth="sizeSidebar10">
```

Comment:
Curious why you chose to copy paste this and delete the component, rather than update the component and import it into a different location

Reply:
Oversight on my part. I copied the original just to get it in place, but then never refactored. Done now.